Resources/Prototypes/_Shitmed/Species/monkey.yml

```
@@ -0,0 +1,144 @@
+- type: markingPoints
+  id: MobMonkeyMarkingLimits
+  points:
+    Special: # the cat ear joke
+      points: 1 # Corvax-Sponsors
+      required: false
+    HeadTop:
+      points: 1
+      required: false
+    Chest:
+      points: 1
+      required: false
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)